src/components/pages/photos/PhotosPage.js

```diff
@@ -29,7 +30,7 @@ export default class PhotosPage extends Component {
         />
         { /* the li is for us to check clicking on the buttons does change the state */ }
         <li>{this.state.currentRover}</li>
```

Comment:
I assume we don't need these lines any more now that we're actually passing down the `currentRover` state?